test/e2e/autotls/auto_tls_test.go

```diff
@@ -57,6 +59,32 @@ func TestTls(t *testing.T) {
 	t.Run(env.AutoTLSTestName, testAutoTLS)
 }
 
+func TestTlsDisabledWithAnnotation(t *testing.T) {
```

Comment:
Go naming requires this to be called `TestTLS...`

Reply:
There's a comment above:
```
// Technically TLS is proper Go casing for acronyms, but it makes our test naming
// hyphenate between every character, and we hit length problems.
```